config/api.yml

```diff
@@ -620,6 +622,25 @@
         :identifier: instance_guest_restart
       - :name: reset
         :identifier: instance_reset
+    :load_balancers_subcollection_actions:
+      :get:
+      - :name: show
+        :identifier: load_balancer_show
+  :load_balancers:
+    :description: Load Balancers
+    :options:
+    - :collection
+    - :subcollection
+    :verbs: *g
+    :klass: LoadBalancer
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: load_balancer_show_list
+    :resource_actions:
+      :get:
+      - :name: read
+        :identifier: load_balancer_show
```

Comment:
are there plans to add other actions to load_balancers (collection or subcollection) ?